### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by CKBoytGT** [CKBoytGT/challenge-1-accessibility#4]
Alt descriptions are needed for every image on the page in order to meet accessibility standards and acceptance criteria. The current site has no alt text.
